house/grpchandlers.go

```diff
@@ -191,3 +191,12 @@ func (s *Server) UpdateDevice(ctx context.Context, request *pb.Devices) (*pb.Rep
 	}
 	return &pb.Reply{Acknowledged: true}, nil
 }
+
+// UpdateDevice Handler for updating Devices
```

Comment:
warning: comment on exported method Server.ListPeople should be of the form "ListPeople ..." (golint)